src/constants/constants.ts

```diff
@@ -187,3 +188,6 @@ export const azureFunctionsExtensionNotFound = localize('azureFunctionsExtension
 export const installAzureFunction = localize('install', 'Install');
 export const learnMore = localize('learnMore', 'Learn more');
 export const doNotInstall = localize('doNotInstall', 'Do not install');
+export const createProject = localize('createProject', 'Create Azure Function Project');
```

Comment:
Should it always be Azure Functions with the 's'? Just want to make sure we are consistent